### DISCUSSION TITLE
Artison not starting on Pi5 64 bit (bookworm)

### DISCUSSION THREAD

**Discussion by glenmbaker** [artisan-roaster-scope/artisan#1450] [General]
I have a brand new Pi5 with a fresh bookworm install (64 bit). I've downloaded and installed the .deb package for this system, but when I try to start artisan I get ... nothing. Just an immediate return to a shell prompt. I can't see any logfiles anywhere, so in a fit of desparation I added a "set -x" in /usr/bin/artisan. It yields the following:

gbaker@moe:~ $ artisan
+++ dirname /usr/bin/artisan
++ cd /usr/bin
++ pwd
+ __dir=/usr/bin
+ '[' -f /etc/os-release ']'
+ grep Raspbian /etc/os-release
+ '[' 0 -eq 0 ']'
+ unset XDG_CURRENT_DESKTOP
+ exec /usr/bin/env LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libwayland-client.so.0 GDK_BACKEND=x11 PATH=/usr/bin/../share/artisan:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/games:/usr/games QT_PLUGIN_PATH=/usr/bin/../share/artisan/PyQt6/Qt6/plugins LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/gtk-3.0/modules:/usr/bin/../share/artisan: LIBOVERLAY_SCROLLBAR=0 GTK2_RC_FILES=/home/gbaker/.gtkrc-2.0 /usr/bin/../share/artisan/artisan

Right off the bat I notice a reference to /usr/lib/x86_64-linux-gnu/libwayland-client.so.0 which doesn't exist. Any clues as to where I should be looking?

Oh, and it looks like the exit code is 127 (echo $? after trying to launch artisan)

**Comment by glenmbaker** [comment]
Huh. I did the obvious (since I really didn't have much on the Pi): re-flashed the OS, re-installed artisan, and presto! it's now working. Clearly user error, but it sure seems like I just did the same thing twice and it worked the second time :)

**Reply by hermetheuscoffee** [reply]
I found this post because I had the EXACT same issue... also on a brand new Pi 5 with a fresh 64-bit Bookworm install. I followed your solution for the fix, and it also fixed it for me! :-) I concur that this doesn't make any sense, because I also felt like I did the same thing twice, but in the end, it worked!
